litho-it/src/test/java/com/facebook/litho/widget/RecyclerBinderTest.java

```diff
@@ -1170,6 +1173,39 @@ public void testInsertMixedContentFollowedByUpdate() {
     assertThat(mRenderInfoViewCreatorController.mViewTypeToViewCreator.size()).isEqualTo(3);
   }
 
+  @Test
+  public void testViewBinderBindAndUnbind() {
+    final View view = mock(View.class);
+    final RecyclerView recyclerView = new RecyclerView(RuntimeEnvironment.application);
```

Comment:
You can use `mComponentContext` instead of `RuntimeEnvironment.application` here.

Reply:
done